components/SpotlightShare/icon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Icon = styled.img`
+  float: left;
+  margin: 0 5px;
+  height: 3em;
+  width: 3em;
+`;
+
+const SVG = (props: any) => (
+  <div>
+    <a href="#">
+      <Icon src={'/assets/svg/source/' + props.socialSource + '.svg'} alt={props.socialSource} />
+    </a>
+  </div>
+);
+
+export { SVG };
```

Comment:
Extract this into its own folder under components, like all the other reusable components ... like you did with SpotlightShare. Also, name it appropriately... SVG is taken, so use something more specific. I suggest `SocialLink`

Reply:
Done